### PR TITLE
Detail model modified & transaction updateStoredDetail added

### PR DESCRIPTION
**Pull request by AngelJSD**

### CODE REVIEW

File: src/core/db-transactions/storagelocation.ts [hunk 1]
```diff
@@ -35,17 +40,20 @@ export function getStorageLocationByCategory(category: string){
 	return deferred.promise;
 }
 
-export function updateAvailableStorageLocation(storageLocationId : string , available : boolean)
+export function updateAvailableStorageLocation(storageLocations : storageLocationObject[])
 {
 	let StorageLocation : MongoModel = new MongoModel('storagelocation');
```
Comment (fabalcu97) [inline src/core/db-transactions/storagelocation.ts:45]: Add exception if storageLocations is empty

Comment (AngelJSD) [reply src/core/db-transactions/storagelocation.ts:45]: Done

File: src/core/db-transactions/detail.ts [hunk 2]
```diff
@@ -14,7 +14,8 @@ export function registerDetail( DetailData: dbModels.Detail) {
         quantity : DetailData.quantity,
         totalPrice : DetailData.totalPrice,
         totalWeight : DetailData.totalWeight,
-		lotQuantity: DetailData.lotQuantity
+		lotQuantity: DetailData.lotQuantity,
+		stored : false
 	};
```
Comment (fabalcu97) [inline src/core/db-transactions/detail.ts:19]: Fix indentation here

File: src/server/routers/api.ts [hunk 3]
```diff
@@ -84,6 +84,20 @@ apiRoutes.addRoute('GET', '/get/detail/:billId', (req, res) => {
 	});
 });
 
+apiRoutes.addRoute('POST', '/updateStored/detail', (req, res) => {
+	
+	Detail.updateStoredDetail(req.body.detailId,req.body.detailStored)
```
Comment (fabalcu97) [inline src/server/routers/api.ts:89]: update this in order to receive the correct params

Comment (AngelJSD) [reply src/server/routers/api.ts:89]: Done

**Review by fabalcu97 — APPROVED** [review]

**Pull request merged**